blaze/server/client.py

```diff
@@ -100,16 +108,18 @@ def compute_down(expr, ec, **kwargs):
     from .server import to_tree
     tree = to_tree(expr)
 
-    r = requests.get('%s/compute.json' % ec.url,
-                     data=json.dumps({'expr': tree}),
-                     headers={'Content-Type': 'application/json'})
+    serial = ec.serial
+    r = requests.get('%s/compute.%s' % (ec.url, ec.mode),
+                     data=serial.dumps({'expr': tree}))
```

Comment:
Thanks, does python2 support the python3 pickle format, I haven't tried, and to remove the issue with qdb I just used json.